src/Proxy/StoredProcedureClients/SerialNumberService.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace Linn.Stores2.Proxy.StoredProcedureClients
+{
+    using System.Data;
+    using System.Threading.Tasks;
+    using Linn.Common.Domain;
+    using Linn.Common.Proxy.LinnApps;
+    using Linn.Stores2.Domain.LinnApps.External;
+    using Oracle.ManagedDataAccess.Client;
+
+    public class SerialNumberService : ISerialNumberService
+    {
+        public async Task<bool> GetSerialNumbersRequired(string partNumber)
+        {
+            using var connection = new OracleConnection(ConnectionStrings.ManagedConnectionString());
+            {
```

Comment:
not 100% sure but  I think if you use a using declaration (with a semi-colon on the end) you don't need a curly bracket block scope after it